src/components/RedeemCode/RedeemCodeCard.tsx

```diff
@@ -1,12 +1,19 @@
-import { Button, Group, Stack, Text, TextInput, ThemeIcon, Loader } from '@mantine/core';
+import { Button, Group, Stack, Text, TextInput, ThemeIcon, Modal } from '@mantine/core';
 import { IconTicket } from '@tabler/icons-react';
+import clsx from 'clsx';
 import React, { useState } from 'react';
-import { showNotification } from '@mantine/notifications';
+import dynamic from 'next/dynamic';
 import { trpc } from '~/utils/trpc';
 import { numberWithCommas } from '~/utils/number-helpers';
 import classes from './RedeemCodeCard.module.scss';
 import { RedeemableCodeType } from '~/shared/utils/prisma/enums';
 import type { SubscriptionProductMetadata } from '~/server/schema/subscriptions.schema';
+import { showErrorNotification, showWarningNotification } from '~/utils/notifications';
+
+const SuccessAnimation = dynamic(
+  () => import('~/components/Animations/SuccessAnimation').then((mod) => mod.SuccessAnimation),
```

Comment:
Not 100% there's any advantage to having this be imported async 🤷

Reply:
I just followed how we do it in other places 🤷